relations/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "relations"
+version = "0.1.0"
+edition = "2021"
+license = "Apache 2.0"
+authors = ["Cardinal Cryptography"]
+
+[dependencies]
+ark-bls12-381 = { version = "^0.3.0" }
+ark-crypto-primitives = { version = "^0.3.0", default-features = true, features = [ "r1cs" ] }
+ark-ec = { version = "^0.3.0", default-features = false }
+ark-ed-on-bls12-381 = { version = "^0.3.0", features = ["r1cs"] }
+ark-ff = { version = "^0.3.0", default-features = false }
+ark-gm17 = { version = "^0.3.0", default-features = false }
+ark-groth16 = { version = "^0.3.0", default-features = false }
+ark-marlin = { version = "^0.3.0", default-features = false }
+ark-poly = { version = "^0.3.0", default-features = false }
+ark-poly-commit = { version = "^0.3.0", default-features = false }
+ark-r1cs-std = { version = "^0.3.0", default-features = false }
+ark-relations = { version = "^0.3.0", default-features = false }
+ark-serialize = { version = "^0.3.0", default-features = false }
+ark-snark = { version = "^0.3.0", default-features = false }
+ark-std = { version = "^0.3.0", default-features = false }
+blake2 = "0.9.2"
+
+[features]
+default = ["std"]
+std = []
```

Comment:
is there any reason for that feature? eventually, this crate will be used in some contracts, so no-std for sure will be needed, but for other usages (like CLI) I'm not so sure that we will need any std functionality (maybe some (de)serializing, but we don't have it anyway)

Reply:
maybe SC test environments... but we're not using those. Removed